### DISCUSSION TITLE
[Discussion] `LLB` and `PLB` - Commands Description

### DISCUSSION THREAD

**Discussion by FelipeRearden** [TfTHacker/obsidian42-text-transporter#13] [Ideas]
Hello @TfTHacker and Friends!

LLB and PLB creates embed block references: 
- `![[filename#^blockref]]`

This way, I think we should change the command description to emphasize the embed feature 

PLB = Push Line(s) From Another File as a Embed Block

LLB = Pull Line(s) From Another File as a Embed Block

What you think friends ?

**Comment by TfTHacker** [comment]
I like this. I will clean up the text a little based on your recommendations. Basically will use the term block embed when it is a ![[]] link and block reference when it is just [[]].

Thanks for feedback. 

Going to push out a 0.8 beta and these tweeks will be included.

**Reply by FelipeRearden** [reply]
I’m glad to know that you liked. 

Thanks for reading!

I have a long insight to share with and our friends but I’m gonna wait for 0.8.0 to see if I have to update what I’m gonna suggest based on the new version. 

Have a great day!
